### DISCUSSION TITLE
Logo for the project?

### DISCUSSION THREAD

**Discussion by whyboris** [casey/just#1498] [General]
I can't seem to find any official logo for `just` 😅 

I'd love to see a nice logo in my _VS Code_ instead of a generic unknown file format - and it can be added in [vscode-great-icons](https://github.com/EmmanuelBeziat/vscode-great-icons/issues) once we have one 😊 

<img width="92" alt="image" src="https://user-images.githubusercontent.com/17264277/211960481-9c8118e7-4c70-4949-b140-bc8ab5388008.png">

Cheers!

**Comment by casey** [comment]
See #1255, there's some discussion there.
